build/ci-codecheck.yml

```diff
@@ -0,0 +1,42 @@
+name: $(Build.Major).$(Build.Minor).$(date:yyMM).$(DayOfMonth)$(rev:rr)
+
+trigger: none
+
+pr:
+- main
+- feature/*
+- features/*
+- release/*
+
+schedules:
+- cron: "0 9 * * Sat"
+  displayName: 'Build for Component Governance'
+  branches:
+    include:
+    - main
+  always: true
+
+variables:
+  Build.Major: 0
+  Build.Minor: 16
```

Comment:
This is out of date; should it even be included?

Reply:
Removed.